### PR TITLE
migration: Update to check result after migraion and add a func to clean vm

### PR DESCRIPTION
**Pull request by Yingshun**
1. Merge check_result with do_migraion
2. Add a method to clear up vm on a local host and remote host

Test results:
https://github.com/autotest/tp-libvirt/pull/3404#issue-602656760

### CODE REVIEW

**Comment by Yingshun** [comment]
@dzhengfy One of the purposes of this PR is to implement the request from https://github.com/autotest/tp-libvirt/pull/3256#discussion_r570677124 , could you please help review it? Thanks!

**Review by dzhengfy — COMMENTED** [review]
LGTM

**Review by dzhengfy — APPROVED** [review]

File: virttest/migration.py [hunk 1]
```diff
@@ -359,6 +359,9 @@ def do_migration(self, vms, srcuri, desturi, migration_type,
         if not self.RET_MIGRATION and not ignore_status:
             raise exceptions.TestFail()
 
+        logging.debug("Checking migration result...")
```
Comment (kylazhang) [inline virttest/migration.py:362]: I think here better use logging.info

File: virttest/migration.py [hunk 2]
```diff
@@ -374,6 +377,19 @@ def cleanup_dest_vm(self, vm, srcuri, desturi):
         # Set connect uri back to local uri
         vm.connect_uri = srcuri
 
+    def cleanup_vm(self, vm, desturi):
+        """
+        Cleanup migrated vm on remote host and local host
+
+        :param desturi: uri for remote access
```
Comment (kylazhang) [inline virttest/migration.py:384]: Missing vm param docstring

**Review by kylazhang — COMMENTED** [review]
Others are OK for me

File: virttest/migration.py (hunk 1 above)
Comment (Yingshun) [reply virttest/migration.py]: Updated.

File: virttest/migration.py (hunk 2 above)
Comment (Yingshun) [reply virttest/migration.py:385]: Oops, fixed.

**Review by kylazhang — APPROVED** [review]

**Pull request merged**